### PR TITLE
chore: update inline-code to code-200

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Updates inline code in MDX to use the `code-200` style from HDS, bumping the font-size to `16px` at default zoom levels, to match surrounding body copy.

> **Note**: The current style did not use an HDS helper class, but corresponded directly within `code-100`. The previous font-size was `13px` at default zoom levels.

## 📸 Design Screenshots
| `font-size` | Notes |  Screenshot |
| - | - | - |
| `13px`  | `code-100`, current | before-13 |
| `14px` | `code-200`, this PR | after |

## 🧪 Testing

- [ ] Visit a page with inline code
    - Example: [/vault/docs/configuration/seal/alicloudkms][/vault/docs/configuration/seal/alicloudkms], compare to [upstream][upstream/vault/docs/configuration/seal/alicloudkms]
    - Inline code should have a font-size of `14px` at the default zoom level, corresponding to [the `code-200` typography style](https://design-system-components-hashicorp.vercel.app/foundations/typography#showcase) from HDS

> **Note**: Inline code in headings should not be affected. We have separate sizing logic for those cases, which is based on the size of the surrounding heading.

## 💭 Anything else?

Not at the moment!

[preview]: https://dev-portal-git-zsmdx-inline-code-size-hashicorp.vercel.app
[task]: https://app.asana.com/0/1202097197789424/1203412239936441/f
[/vault/docs/configuration/seal/alicloudkms]: https://dev-portal-git-zsmdx-inline-code-size-hashicorp.vercel.app/vault/docs/configuration/seal/alicloudkms
[upstream/vault/docs/configuration/seal/alicloudkms]: https://developer.hashicorp.com/vault/docs/configuration/seal/alicloudkms

### CODE REVIEW

**Comment by braydenlove** [comment]
@zchsh awesome, it is so great to see these in practice. A few things...
1. The padding around the code should be `padding: 2px 4px 3px 4px;` to account for the code-link styling.
2. `code-200` (according to HDS) should be `font-weight: 400; font-size: 14px; line-height: 18px;`. Here is the [link](https://design-system-components-hashicorp.vercel.app/foundations/tokens#css-custom-properties) to the HDS docs. You have to scroll to the bottom to see the `code-300` and `code-200` css!

**Comment by zchsh** [comment]
@braydenlove Ah nice, thanks for catching that - have updated the padding, and backfilled the previously missing `hds-typography-code-200` tokens & helper classes, so once the preview builds this should be properly up to date and show the correct `font-size`, `font-weight`, and `line-height` 👍

**Review by kendallstrautman — APPROVED** [review]

File: src/components/dev-dot-content/mdx-components/mdx-inline-code/mdx-inline-code.module.css [hunk 1]
```diff
@@ -1,9 +1,23 @@
+/* Note: https://github.com/hashicorp/design-system-tokens has been marked
```
[the remark this answers is not in the document]
Comment (braydenlove) [reply src/components/dev-dot-content/mdx-components/mdx-inline-code/mdx-inline-code.module.css:1]: I just heard back from HDS and I guess everything got merged in the monorepo [here](https://github.com/hashicorp/design-system). Also, [here](https://github.com/hashicorp/design-system/tree/main/packages/tokens/dist/devdot/css) are the generated tokens if helpful.

[the remark this answers is not in the document]
Comment (braydenlove) [reply src/components/dev-dot-content/mdx-components/mdx-inline-code/mdx-inline-code.module.css:1]:
```
  --token-typography-code-100-font-family: ui-monospace, Menlo, Consolas, monospace;
  --token-typography-code-100-font-size: 0.8125rem; /* 13px/0.8125rem */
  --token-typography-code-100-line-height: 1.23; /* 16px */
  --token-typography-code-200-font-family: ui-monospace, Menlo, Consolas, monospace;
  --token-typography-code-200-font-size: 0.875rem; /* 14px/0.875rem */
  --token-typography-code-200-line-height: 1.125; /* 18px */
  --token-typography-code-300-font-family: ui-monospace, Menlo, Consolas, monospace;
  --token-typography-code-300-font-size: 1rem; /* 16px/1rem */
  --token-typography-code-300-line-height: 1.25; /* 20px */
```

[the remark this answers is not in the document]
Comment (kendallstrautman) [reply src/components/dev-dot-content/mdx-components/mdx-inline-code/mdx-inline-code.module.css:1]: @braydenlove so should we adjust to use the package or does the package not have the code styles we need yet?

[the remark this answers is not in the document]
Comment (zchsh) [reply src/components/dev-dot-content/mdx-components/mdx-inline-code/mdx-inline-code.module.css]: ✅ Have bumped to the latest package, which has these tokens! No real problem here, we just fell behind a bit is all 👍

**Comment by zchsh** [comment]
Have bumped `@hashicorp/design-systems-tokens`, which removes the crufty stopgap I had in place. Have confirmed locally that the resulting CSS values are identical to when this PR was approved. Going to merge this in once the build is done & I can double-check on the preview 👍

**Pull request merged**